src/CentralDesktop/Stomp/Connection.php

```diff
@@ -620,12 +622,13 @@ function _writeFrame(Frame $stompFrame, $reconnect = true) {
         $data = $stompFrame->__toString();
 
         $this->logger->debug("Sending Frame", array('frame' => $data));
-        $r = fwrite($this->_socket, $data, strlen($data));
+        $r = fputs($this->_socket, $data, strlen($data));
```

Comment:
I also see a bug here that I thought we fixed.  Is this diff out of date?  I'll check.

strlen($data) will destroy data if there are multibyte characters.